### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by DariaVeretyak**
http://dariaveretyak.github.io/react_article/

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,15 +1,22 @@
 import React from 'react';
+import Article from './components/Article';
+
+const App = () => (
+  <div>
```
Comment (ArtemTopolskyi) [inline src/App.js:5]: do you need this div?

File: src/App.js [hunk 2]
```diff
@@ -1,15 +1,22 @@
...
+    <Article
+      title={article.title}
+      text={article.text}
+      date={article.date}
+    />
+
+    <Article />
```
Comment (ArtemTopolskyi) [inline src/App.js:12]:
```suggestion
    <Article
      title={article.title}
      text={article.text}
      date={article.date}
    />
```

**Review by ArtemTopolskyi — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (vitaliy-martynyuk) [inline src/App.js:5]: if you are using a wrapper for more than 1 component, it is better to use `<fragment></fragment>` or its' shortened version - `<></>`

**Review by vitaliy-martynyuk — CHANGES_REQUESTED** [review]

**Review by vitaliy-martynyuk — CHANGES_REQUESTED** [review]
if you made changes, you've probably forgotten to commit and push them to your repository

**Comment by DariaVeretyak** [comment]
Can you tell me, please, how to fix this error? Сan't attach screenshot


remote: Permission to mate-academy/react_article.git denied to DariaVeretyak.
fatal: unable to access 'https://github.com/mate-academy/react_article.git/': The requested URL returned error: 403

**Pull request closed without merging**